conan/api/model/list.py

```diff
@@ -36,6 +36,10 @@ def merge(self, other):
         for k, v in other.lists.items():
             self.lists.setdefault(k, PackagesList()).merge(v)
 
+    def keep_outer(self, other):
```

Comment:
If you suggest another name, I'm all ears 😁

Reply:
Reads like a nice name to me